src/NuGet.Core/NuGet.Protocol/Plugins/PluginFactory.cs

```diff
@@ -426,6 +441,20 @@ private void UnregisterEventHandlers(Plugin plugin)
             }
         }
 
+        private static int? GetExitCodeOrNull(Process process)
```

Comment:
Extension method instead?

Reply:
If there were a general need for this in the future, maybe.  Not now.